### PR TITLE
Unfreeze and add SpearofLonginus.netkan

### PR DESCRIPTION
**Pull request by satoi8080**

### CODE REVIEW

**Comment by linuxgurugamer** [comment]
I've notified the author that it needs a release file (notified on Github and in the forums)

**Comment by linuxgurugamer** [comment]
#rebuild

File: NetKAN/SpearofLonginus.netkan [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+     "spec_version" : "v1.4",
+     "identifier"   : "SpearofLonginus",
+     "name"         : "Spear of Longinus",
+     "version"      : "0.8",
```
Comment (Olympic1) [inline NetKAN/SpearofLonginus.netkan]: This will give problems for future releases. as every future version will be 0.8

**Comment by Olympic1** [comment]
Merged with #5527

**Pull request closed without merging**